prometheus-operator.yaml

```diff
@@ -86,6 +86,23 @@ subpackages:
             admission-webhook --version
             admission-webhook --help
 
+  - name: ${{package.name}}-bitnami-compat
+    dependencies:
+      runtime:
+        - prometheus-operator
+        - prometheus-config-reloader
+    description: "compat package with bitnami/prometheus-operator image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: prometheus-operator
+          version-path: 0/debian-12
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/${{package.name}}/bin/
+          chmod g+rwX ${{targets.subpkgdir}}/opt/bitnami
+          ln -sf /usr/bin/prometheus-operator ${{targets.subpkgdir}}/opt/bitnami/${{package.name}}/bin/prometheus-operator
+          ln -sf /usr/bin/prometheus-config-reloader ${{targets.subpkgdir}}/opt/bitnami/${{package.name}}/bin/prometheus-config-reloader
+
```

Comment:
Missing tests, you can take a look at some example like the ingress-nginx-controller compat package [here](https://github.com/wolfi-dev/os/pull/43974/files#diff-52aa8417bde0a0fafd1ee02666798aef371e0940757cf30b8d8850b68fdbd513R542)